packages/components/datagrid/src/js/datagrid.controller.js

```diff
@@ -142,6 +142,16 @@ export default class DatagridController {
       this.paginationMode = paginationMode;
       this.paging.paginationMode = paginationMode;
     }
+
+    if (changes.page && !changes.page.isFirstChange()) {
```

Comment:
**question:** Can you explain why do we need `changes.page.isFirstChange()`?

Reply:
Hello,
Because all of the instantiations are done in the $postLink hook, there is no need to react to the first change :
https://github.com/ovh/ovh-ui-kit/blob/f14f8a89cd09a5d1d7de63c3fcfde1f66eace7b9/packages/components/datagrid/src/js/datagrid.controller.js#L64
And basically all the properties react the same way :
https://github.com/ovh/ovh-ui-kit/blob/f14f8a89cd09a5d1d7de63c3fcfde1f66eace7b9/packages/components/datagrid/src/js/datagrid.controller.js#L126